pkg/metrics/metrics.jsx

```diff
@@ -1789,8 +1805,10 @@ class MetricsHistory extends React.Component {
                 this.setState({
                     loading: false,
                     metricsAvailable: false,
+                    isPythonPCPInstalled: message?.message !== "python3-pcp not installed",
```

Comment:
Please do `message.problem === "not-supported"` ? That's much better than this human string comparison. Also, your mock hack just does `raise ChannelError('not-supported')` without a message.

(The `?` here is unnecessary -- you already asserted that message.problem exists)

Reply:
No, the mock is for a different scenario.